### ISSUE TITLE
[ChannelTabs] Request: Add an option to  revert to the old UI

### ISSUE THREAD

**Issue by HRudyPlayZ** [l0c4lh057/BetterDiscordStuff#92]
**Is your feature request related to a problem? Please describe.**
Some people might not like the new UI /  People with larger displays will struggle more to click on the close button.

**Describe the solution you'd like**
An option to revert to the old UI.  That and fix the theme compatibility, but that's probably planned.

**Describe alternatives you've considered**
Another way would be to bring the close button to the tab center, make it bigger, white instead of red, make the tabs slightly longer for the indicators and revert to the old tab shape (i'm not talking about the inactive tabs not having a background, but the general shape of tabs when selected)

**Additional context**
The old UI for reference:
![image](https://user-images.githubusercontent.com/29356467/127405907-287c61d7-a1c5-4df7-be21-a7a68cf98f95.png)

**Comment by l0c4lh057** [comment]
there is a compact mode in the setting, i think it doesnt look exactly the same but similar

**Comment by HRudyPlayZ** [comment]
Agreed, i also think compact mode should be the default. But it doesn't fix everything.

**Comment by HRudyPlayZ** [comment]
One other thing that could be done (could also be a separated issue) would be to allow to open the quick settings when right clicking on the empty tab bar.

**Comment by KaKi87** [comment]
> compact mode should be the default

Actually, the user-friendly way would be to enable compact mode by default only if the [native compact mode](https://support.discord.com/hc/en-us/articles/217047657-How-do-I-switch-to-compact-text-mode-) is enabled as well.
